### DISCUSSION TITLE
Git Premature EOF

### DISCUSSION THREAD

**Discussion by laurent-madarassou** [apache/netbeans#7416] [General]
### Apache NetBeans version

Apache NetBeans 21

### What happened

 Hi team.

I'm trying to clone a project from GitHub repository.But a few seconds (about 40 secondes) i get an 'Premature EOF' error

 
![NetBeansGitPrematureEOF](https://github.com/apache/netbeans/assets/93809851/96bec389-fbff-4c40-a495-c5f07dc026d4)

Have you ever seen this issue ?
I had tried every thing but i did not succeed anyway .

Thanks for your answer.

### Language / Project Type / NetBeans Component

Java project on Github. 

### How to reproduce

At this time, i can't exactly tell you how i did this issue. I do really think it deals with 'Broken Upload Repository'  ... but i don't have more explanations about this .

Thanks for your answers.

### Did this work correctly in an earlier version?

Apache NetBeans 21

### Operating System

Linux Mint 21

### JDK

11

### Apache NetBeans packaging

Apache NetBeans Snap Package

### Are you willing to submit a pull request?

Yes

**Comment by matthiasblaesing** [comment]
Please check the message log/IDE Log. In the IDE you can access it via View -> IDE Log. For the log file here are hints how to find it:

https://stackoverflow.com/questions/50462124/netbeans-where-to-find-the-ide-log
